### ISSUE TITLE
Cannot run the example

### ISSUE THREAD

**Issue by OmarAshkar** [DeepPSP/cpsc2021#7]
Hello, 
I am new to ECG and I was trying to run the example here as it seems this is a significant changes between current torch_ecg and the way it was implemented here. I am trying to run the jupyter notebook.

I installed latest version of torch_ecg. The implementation here calls torch_ecg.torch_ecg, which I assume was submodule in the repo. 

I fixed that to torch_ecg, but still most of the functions seems to be renamed, so there is no `resnet`  or `resnet_ledwise` anymore!

I would appreciate if there is a working example for this dataset.

Thank you!

**Comment by wenh06** [comment]
Yes, it has changed (evolved) a lot since the last commit of this repo. You can check the latest version at https://github.com/DeepPSP/torch_ecg/tree/master/benchmarks/train_hybrid_cpsc2021

If you have any more questions, please don't hesitate to ask.
